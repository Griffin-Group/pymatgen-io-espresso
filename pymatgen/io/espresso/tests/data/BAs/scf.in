&control
  calculation = 'scf'
  restart_mode = 'from_scratch'
  prefix = 'x'
  pseudo_dir = '.'
  outdir = './out'
  verbosity = 'high'
  wf_collect = .true.
  max_seconds = 900
/

&system
  ibrav = 0
  celldm(1) = 1.889725989d0
  ecutwfc = 60.0D0
  noncolin = .false.
  lspinorb = .false.
  occupations = 'smearing'
  smearing = 'gaussian'
  degauss = 0.001
  nat = 2
  ntyp = 2
/

&electrons
  diagonalization = 'david'
  conv_thr = 1.0D-7
/

&ions
  ion_dynamics = 'bfgs'
/


ATOMIC_SPECIES
B  1.0  B.upf
As  1.0  As.upf

CELL_PARAMETERS alat
  0.0  2.409627  2.409627
  2.409627  0.0  2.409627
  2.409627  2.409627  0.0

ATOMIC_POSITIONS angstrom 
B  3.6144404999999997  3.6144404999999997  3.6144404999999997
As  0.0  0.0  0.0

K_POINTS automatic 
4 4 4 0 0 0 
