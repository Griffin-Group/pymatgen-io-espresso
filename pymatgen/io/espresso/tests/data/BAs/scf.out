
     Program PWSCF v.7.1 starts on 23Jun2023 at 16:30:48 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on      32 processor cores
     Number of MPI processes:                32
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     499017 MiB available memory on the printing compute node when the environment starts

     Reading input from pw.in

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4
     Message from routine cell_base_init:
     DEPRECATED: use true lattice parameter, not A to a.u. conversion factor
     Message from routine setup:
     using ibrav=0 with symmetry is DISCOURAGED, use correct ibrav instead

     K-points division:     npool     =       2
     R & G space division:  proc/nbgrp/npool/nimage =      16
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          42      42     13                  745      745     123
     Max          43      43     14                  749      749     126
     Sum         685     685    211                11935    11935    1989

     Using Slab Decomposition



     bravais-lattice index     =            0
     lattice parameter (alat)  =       1.8897  a.u.
     unit-cell volume          =     188.8322 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            2
     number of electrons       =        18.00
     number of Kohn-Sham states=           13
     kinetic-energy cutoff     =      60.0000  Ry
     charge density cutoff     =     240.0000  Ry
     scf convergence threshold =      1.0E-07
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     celldm(1)=   1.889726  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   0.000000   2.409627   2.409627 )  
               a(2) = (   2.409627   0.000000   2.409627 )  
               a(3) = (   2.409627   2.409627   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -0.207501  0.207501  0.207501 )  
               b(2) = (  0.207501 -0.207501  0.207501 )  
               b(3) = (  0.207501  0.207501 -0.207501 )  


     PseudoPot. # 1 for B  read from file:
     ./B.upf
     MD5 check sum: 72f82fc31bc892420d0cab7e25c88ab8
     Pseudo is Norm-conserving + core correction, Zval =  3.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1534 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1

     PseudoPot. # 2 for As read from file:
     ./As.upf
     MD5 check sum: bfe1a874457ce592347346129958e7cf
     Pseudo is Norm-conserving + core correction, Zval = 15.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1358 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2

     atomic species   valence    mass     pseudopotential
        B              3.00     1.00000     B ( 1.00)
        As            15.00     1.00000     As( 1.00)

     24 Sym. Ops. (no inversion) found


                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000 -0.0000000  0.0000000 )
                  ( -0.0000000  1.0000000  0.0000000 )
                  ( -0.0000000  0.0000000  1.0000000 )


      isym =  2     180 deg rotation - cart. axis [0,0,1]        

 cryst.   s( 2) = (     0          1         -1      )
                  (     1          0         -1      )
                  (     0          0         -1      )

 cart.    s( 2) = ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )
                  (  0.0000000 -0.0000000  1.0000000 )


      isym =  3     180 deg rotation - cart. axis [0,1,0]        

 cryst.   s( 3) = (     0         -1          1      )
                  (     0         -1          0      )
                  (     1         -1          0      )

 cart.    s( 3) = ( -1.0000000 -0.0000000 -0.0000000 )
                  (  0.0000000  1.0000000 -0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym =  4     180 deg rotation - cart. axis [1,0,0]        

 cryst.   s( 4) = (    -1          0          0      )
                  (    -1          0          1      )
                  (    -1          1          0      )

 cart.    s( 4) = (  1.0000000  0.0000000 -0.0000000 )
                  ( -0.0000000 -1.0000000 -0.0000000 )
                  ( -0.0000000 -0.0000000 -1.0000000 )


      isym =  5     120 deg rotation - cart. axis [-1,-1,-1]     

 cryst.   s( 5) = (     0          0          1      )
                  (     1          0          0      )
                  (     0          1          0      )

 cart.    s( 5) = (  0.0000000  1.0000000 -0.0000000 )
                  (  0.0000000 -0.0000000  1.0000000 )
                  (  1.0000000 -0.0000000  0.0000000 )


      isym =  6     120 deg rotation - cart. axis [-1,1,1]       

 cryst.   s( 6) = (     1         -1          0      )
                  (     0         -1          1      )
                  (     0         -1          0      )

 cart.    s( 6) = ( -0.0000000 -1.0000000 -0.0000000 )
                  ( -0.0000000  0.0000000  1.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )


      isym =  7     120 deg rotation - cart. axis [1,1,-1]       

 cryst.   s( 7) = (    -1          1          0      )
                  (    -1          0          0      )
                  (    -1          0          1      )

 cart.    s( 7) = ( -0.0000000  1.0000000  0.0000000 )
                  ( -0.0000000 -0.0000000 -1.0000000 )
                  ( -1.0000000 -0.0000000 -0.0000000 )


      isym =  8     120 deg rotation - cart. axis [1,-1,1]       

 cryst.   s( 8) = (     0          0         -1      )
                  (     0          1         -1      )
                  (     1          0         -1      )

 cart.    s( 8) = (  0.0000000 -1.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )
                  (  1.0000000  0.0000000 -0.0000000 )


      isym =  9     120 deg rotation - cart. axis [1,1,1]        

 cryst.   s( 9) = (     0          1          0      )
                  (     0          0          1      )
                  (     1          0          0      )

 cart.    s( 9) = ( -0.0000000  0.0000000  1.0000000 )
                  (  1.0000000  0.0000000 -0.0000000 )
                  (  0.0000000  1.0000000 -0.0000000 )


      isym = 10     120 deg rotation - cart. axis [-1,1,-1]      

 cryst.   s(10) = (    -1          0          1      )
                  (    -1          1          0      )
                  (    -1          0          0      )

 cart.    s(10) = (  0.0000000 -0.0000000  1.0000000 )
                  ( -1.0000000 -0.0000000 -0.0000000 )
                  ( -0.0000000 -1.0000000 -0.0000000 )


      isym = 11     120 deg rotation - cart. axis [1,-1,-1]      

 cryst.   s(11) = (     1          0         -1      )
                  (     0          0         -1      )
                  (     0          1         -1      )

 cart.    s(11) = (  0.0000000  0.0000000 -1.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )
                  ( -0.0000000  1.0000000  0.0000000 )


      isym = 12     120 deg rotation - cart. axis [-1,-1,1]      

 cryst.   s(12) = (     0         -1          0      )
                  (     1         -1          0      )
                  (     0         -1          1      )

 cart.    s(12) = ( -0.0000000 -0.0000000 -1.0000000 )
                  (  1.0000000 -0.0000000  0.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )


      isym = 13     inv. 180 deg rotation - cart. axis [1,1,0]   

 cryst.   s(13) = (     1          0         -1      )
                  (     0          1         -1      )
                  (     0          0         -1      )

 cart.    s(13) = (  0.0000000 -1.0000000  0.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )
                  ( -0.0000000  0.0000000  1.0000000 )


      isym = 14     inv. 180 deg rotation - cart. axis [1,-1,0]  

 cryst.   s(14) = (     0          1          0      )
                  (     1          0          0      )
                  (     0          0          1      )

 cart.    s(14) = ( -0.0000000  1.0000000  0.0000000 )
                  (  1.0000000 -0.0000000  0.0000000 )
                  (  0.0000000 -0.0000000  1.0000000 )


      isym = 15     inv.  90 deg rotation - cart. axis [0,0,-1]  

 cryst.   s(15) = (     0         -1          0      )
                  (     0         -1          1      )
                  (     1         -1          0      )

 cart.    s(15) = ( -0.0000000 -1.0000000 -0.0000000 )
                  (  1.0000000  0.0000000 -0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym = 16     inv.  90 deg rotation - cart. axis [0,0,1]   

 cryst.   s(16) = (    -1          0          1      )
                  (    -1          0          0      )
                  (    -1          1          0      )

 cart.    s(16) = (  0.0000000  1.0000000 -0.0000000 )
                  ( -1.0000000 -0.0000000 -0.0000000 )
                  ( -0.0000000 -0.0000000 -1.0000000 )


      isym = 17     inv. 180 deg rotation - cart. axis [1,0,1]   

 cryst.   s(17) = (     1         -1          0      )
                  (     0         -1          0      )
                  (     0         -1          1      )

 cart.    s(17) = ( -0.0000000 -0.0000000 -1.0000000 )
                  ( -0.0000000  1.0000000  0.0000000 )
                  ( -1.0000000  0.0000000  0.0000000 )


      isym = 18     inv. 180 deg rotation - cart. axis [-1,0,1]  

 cryst.   s(18) = (     0          0          1      )
                  (     0          1          0      )
                  (     1          0          0      )

 cart.    s(18) = (  0.0000000 -0.0000000  1.0000000 )
                  (  0.0000000  1.0000000 -0.0000000 )
                  (  1.0000000  0.0000000 -0.0000000 )


      isym = 19     inv.  90 deg rotation - cart. axis [0,1,0]   

 cryst.   s(19) = (     0          0         -1      )
                  (     1          0         -1      )
                  (     0          1         -1      )

 cart.    s(19) = (  0.0000000  0.0000000 -1.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )
                  (  1.0000000 -0.0000000  0.0000000 )


      isym = 20     inv.  90 deg rotation - cart. axis [0,-1,0]  

 cryst.   s(20) = (    -1          1          0      )
                  (    -1          0          1      )
                  (    -1          0          0      )

 cart.    s(20) = ( -0.0000000  0.0000000  1.0000000 )
                  ( -0.0000000 -1.0000000 -0.0000000 )
                  ( -1.0000000 -0.0000000 -0.0000000 )


      isym = 21     inv. 180 deg rotation - cart. axis [0,1,1]   

 cryst.   s(21) = (    -1          0          0      )
                  (    -1          1          0      )
                  (    -1          0          1      )

 cart.    s(21) = (  1.0000000 -0.0000000  0.0000000 )
                  ( -0.0000000 -0.0000000 -1.0000000 )
                  ( -0.0000000 -1.0000000 -0.0000000 )


      isym = 22     inv. 180 deg rotation - cart. axis [0,1,-1]  

 cryst.   s(22) = (     1          0          0      )
                  (     0          0          1      )
                  (     0          1          0      )

 cart.    s(22) = (  1.0000000  0.0000000 -0.0000000 )
                  ( -0.0000000  0.0000000  1.0000000 )
                  ( -0.0000000  1.0000000  0.0000000 )


      isym = 23     inv.  90 deg rotation - cart. axis [-1,0,0]  

 cryst.   s(23) = (     0          1         -1      )
                  (     0          0         -1      )
                  (     1          0         -1      )

 cart.    s(23) = ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )
                  (  0.0000000  1.0000000 -0.0000000 )


      isym = 24     inv.  90 deg rotation - cart. axis [1,0,0]   

 cryst.   s(24) = (     0         -1          1      )
                  (     1         -1          0      )
                  (     0         -1          0      )

 cart.    s(24) = ( -1.0000000 -0.0000000 -0.0000000 )
                  (  0.0000000 -0.0000000  1.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )


   Cartesian axes

     site n.     atom                  positions (alat units)
         1           B   tau(   1) = (   3.6144408   3.6144408   3.6144408  )
         2           As  tau(   2) = (   0.0000000   0.0000000   0.0000000  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           B   tau(   1) = (  0.7500001  0.7500001  0.7500001  )
         2           As  tau(   2) = (  0.0000000  0.0000000  0.0000000  )

     number of k points=     8  Gaussian smearing, width (Ry)=  0.0010
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0312500
        k(    2) = (   0.0518752   0.0518752  -0.0518752), wk =   0.2500000
        k(    3) = (  -0.1037505  -0.1037505   0.1037505), wk =   0.1250000
        k(    4) = (   0.1037505   0.0000000   0.0000000), wk =   0.1875000
        k(    5) = (  -0.0518752  -0.1556257   0.1556257), wk =   0.7500000
        k(    6) = (   0.0000000  -0.1037505   0.1037505), wk =   0.3750000
        k(    7) = (  -0.2075010   0.0000000   0.0000000), wk =   0.0937500
        k(    8) = (  -0.2075010   0.1037505   0.0000000), wk =   0.1875000

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0312500
        k(    2) = (   0.0000000   0.0000000   0.2500000), wk =   0.2500000
        k(    3) = (  -0.0000000  -0.0000000  -0.5000000), wk =   0.1250000
        k(    4) = (   0.0000000   0.2500000   0.2500000), wk =   0.1875000
        k(    5) = (  -0.0000000   0.2500000  -0.5000000), wk =   0.7500000
        k(    6) = (  -0.0000000   0.2500000  -0.2500000), wk =   0.3750000
        k(    7) = (   0.0000000  -0.5000000  -0.5000000), wk =   0.0937500
        k(    8) = (   0.2500000  -0.5000000  -0.2500000), wk =   0.1875000

     Dense  grid:    11935 G-vectors     FFT dimensions: (  32,  32,  32)

     Dynamical RAM for                 wfc:       0.02 MB

     Dynamical RAM for     wfc (w. buffer):       0.09 MB

     Dynamical RAM for           str. fact:       0.02 MB

     Dynamical RAM for           local pot:       0.00 MB

     Dynamical RAM for          nlocal pot:       0.04 MB

     Dynamical RAM for                qrad:       2.49 MB

     Dynamical RAM for          rho,v,vnew:       0.08 MB

     Dynamical RAM for               rhoin:       0.03 MB

     Dynamical RAM for            rho*nmix:       0.18 MB

     Dynamical RAM for           G-vectors:       0.05 MB

     Dynamical RAM for          h,s,v(r/c):       0.03 MB

     Dynamical RAM for          <psi|beta>:       0.01 MB

     Dynamical RAM for                 psi:       0.04 MB

     Dynamical RAM for                hpsi:       0.04 MB

     Dynamical RAM for      wfcinit/wfcrot:       0.04 MB

     Estimated static dynamical RAM per process >       2.91 MB

     Estimated max dynamical RAM per process >       3.20 MB

     Estimated total dynamical RAM >      99.44 MB

     Initial potential from superposition of free atoms

     starting charge      17.9995, renormalised to      18.0000
     Starting wfcs are   13 randomized atomic wfcs

     total cpu time spent up to now is        0.4 secs

     Self-consistent Calculation

     iteration #  1     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
            46 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
        498675 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.00E-02,  avg # of iterations =  3.9

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
            47 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
        498639 MiB available memory on the node where the printing process lives
------------------
     ethr =  8.71E-04,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.4 secs

     total energy              =    -217.82296231 Ry
     estimated scf accuracy    <       0.15908383 Ry

     iteration #  2     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
            48 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
        498639 MiB available memory on the node where the printing process lives
------------------
     ethr =  8.84E-04,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.4 secs

     total energy              =    -217.85811094 Ry
     estimated scf accuracy    <       0.05301767 Ry

     iteration #  3     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
            48 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
        498639 MiB available memory on the node where the printing process lives
------------------
     ethr =  2.95E-04,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.5 secs

     total energy              =    -217.86882509 Ry
     estimated scf accuracy    <       0.00065448 Ry

     iteration #  4     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
            48 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
        498639 MiB available memory on the node where the printing process lives
------------------
     ethr =  3.64E-06,  avg # of iterations =  3.5

     total cpu time spent up to now is        0.5 secs

     total energy              =    -217.86891672 Ry
     estimated scf accuracy    <       0.00002914 Ry

     iteration #  5     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
            48 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
        498640 MiB available memory on the node where the printing process lives
------------------
     ethr =  1.62E-07,  avg # of iterations =  2.4

     total cpu time spent up to now is        0.5 secs

     total energy              =    -217.86892391 Ry
     estimated scf accuracy    <       0.00000044 Ry

     iteration #  6     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap

---- Real-time Memory Report at c_bands before calling an iterative solver
            49 MiB given to the printing process from OS
             0 MiB allocation reported by mallinfo(arena+hblkhd)
        498640 MiB available memory on the node where the printing process lives
------------------
     ethr =  2.45E-09,  avg # of iterations =  3.5

     total cpu time spent up to now is        0.5 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (  1459 PWs)   bands (ev):

   -25.1393 -25.1393 -25.1393 -25.1334 -25.1334  -4.2145  10.9712  10.9712
    10.9712  14.2195  14.2195  14.2195  15.3006

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000   0.0000

          k = 0.0519 0.0519-0.0519 (  1468 PWs)   bands (ev):

   -25.1432 -25.1404 -25.1404 -25.1144 -25.1144  -3.2399   6.1053   9.8470
     9.8470  13.9016  15.5809  15.5810  18.6981

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000   0.0000

          k =-0.1038-0.1038 0.1038 (  1476 PWs)   bands (ev):

   -25.1619 -25.1469 -25.1469 -25.1197 -25.1197  -1.4443   2.3680   9.1864
     9.1864  13.5142  15.7494  15.7494  19.8262

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000   0.0000

          k = 0.1038 0.0000 0.0000 (  1475 PWs)   bands (ev):

   -25.1510 -25.1510 -25.1437 -25.1221 -25.1182  -2.9039   6.8727   8.3342
     8.3342  12.6568  15.4650  18.2446  18.2446

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000   0.0000

          k =-0.0519-0.1556 0.1556 (  1478 PWs)   bands (ev):

   -25.1574 -25.1411 -25.1342 -25.1245 -25.1188  -1.0448   3.3772   6.3441
     7.8701  13.3858  16.7573  18.5163  18.7942

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000   0.0000

          k = 0.0000-0.1038 0.1038 (  1478 PWs)   bands (ev):

   -25.1538 -25.1470 -25.1315 -25.1273 -25.1224  -1.7707   4.4108   6.0115
     9.0461  14.4493  17.0114  17.8286  18.2422

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000   0.0000

          k =-0.2075 0.0000 0.0000 (  1524 PWs)   bands (ev):

   -25.1893 -25.1893 -25.1822 -25.1736 -25.1618  -0.1790   2.5889   6.9887
     6.9887  12.4469  12.4652  22.1070  22.1070

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000   0.0000

          k =-0.2075 0.1038 0.0000 (  1480 PWs)   bands (ev):

   -25.1493 -25.1353 -25.1335 -25.1302 -25.1219  -0.0332   3.0333   5.6326
     5.9251  16.2664  16.7720  17.7447  17.8703

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000   0.0000

     the Fermi energy is    11.8375 ev

!    total energy              =    -217.86892408 Ry
     estimated scf accuracy    <       0.00000004 Ry
     smearing contrib. (-TS)   =      -0.00000000 Ry
     internal energy E=F+TS    =    -217.86892408 Ry

     The total energy is F=E-TS. E is the sum of the following terms:
     one-electron contribution =    -103.61195246 Ry
     hartree contribution      =      61.37520449 Ry
     xc contribution           =     -49.90216396 Ry
     ewald contribution        =    -125.73001215 Ry

     convergence has been achieved in   6 iterations

     Writing all to output data dir ./out/x.save/

     init_run     :      0.16s CPU      0.18s WALL (       1 calls)
     electrons    :      0.16s CPU      0.19s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.01s CPU      0.03s WALL (       1 calls)
     wfcinit:atom :      0.00s CPU      0.00s WALL (       4 calls)
     wfcinit:wfcr :      0.01s CPU      0.03s WALL (       4 calls)
     potinit      :      0.01s CPU      0.01s WALL (       1 calls)
     hinit0       :      0.13s CPU      0.14s WALL (       1 calls)

     Called by electrons:
     c_bands      :      0.12s CPU      0.15s WALL (       7 calls)
     sum_band     :      0.02s CPU      0.02s WALL (       7 calls)
     v_of_rho     :      0.01s CPU      0.01s WALL (       7 calls)
     v_h          :      0.00s CPU      0.00s WALL (       7 calls)
     v_xc         :      0.01s CPU      0.01s WALL (       7 calls)
     mix_rho      :      0.00s CPU      0.00s WALL (       7 calls)

     Called by c_bands:
     init_us_2    :      0.00s CPU      0.00s WALL (      60 calls)
     init_us_2:cp :      0.00s CPU      0.00s WALL (      60 calls)
     cegterg      :      0.11s CPU      0.13s WALL (      28 calls)

     Called by sum_band:
     sum_band:wei :      0.00s CPU      0.00s WALL (       7 calls)
     sum_band:loo :      0.02s CPU      0.02s WALL (       7 calls)
     sum_band:buf :      0.00s CPU      0.00s WALL (      28 calls)
     sum_band:ini :      0.00s CPU      0.00s WALL (      28 calls)

     Called by *egterg:
     cdiaghg      :      0.01s CPU      0.02s WALL (      99 calls)
     cegterg:over :      0.01s CPU      0.01s WALL (      75 calls)
     cegterg:upda :      0.00s CPU      0.00s WALL (      75 calls)
     cegterg:last :      0.00s CPU      0.00s WALL (      55 calls)
     h_psi        :      0.09s CPU      0.11s WALL (     107 calls)
     g_psi        :      0.00s CPU      0.00s WALL (      75 calls)

     Called by h_psi:
     h_psi:calbec :      0.01s CPU      0.01s WALL (     107 calls)
     vloc_psi     :      0.08s CPU      0.10s WALL (     107 calls)
     add_vuspsi   :      0.00s CPU      0.00s WALL (     107 calls)

     General routines
     calbec       :      0.01s CPU      0.01s WALL (     107 calls)
     fft          :      0.13s CPU      0.14s WALL (      71 calls)
     ffts         :      0.00s CPU      0.00s WALL (       7 calls)
     fftw         :      0.10s CPU      0.11s WALL (    2292 calls)

     Parallel routines

     PWSCF        :      0.40s CPU      0.60s WALL


   This run was terminated on:  16:30:49  23Jun2023            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
Note: The following floating-point exceptions are signalling: IEEE_DENORMAL
